### ISSUE TITLE
Not being able to use mount

### ISSUE THREAD

**Issue by franciscocpg** [justwatchcom/gopass#75]
Whenever I try to run `gopass mount` I get `Error: Entry is not in the password store`.
I already have a root store.
What are the steps to add a new store and mount?

**Comment by franciscocpg** [comment]
I've already read README and didn't help me either

**Comment by dominikschulz** [comment]
Sounds like a typo in the docs, it's `gopass mounts` not `gopass mount`.

We'll double check a improve the docs if necessary.

**Comment by franciscocpg** [comment]
I'm still confused, :confused: 
I am trying to add another store and here are the steps I've tried:

1. Create a new git repo
2. Run `gopass clone git@myhost:myteam/myrepo.git test`. At this step the clone is successfully completed and then I got an error `Error: password store $HOME/.password-store-test is not initialized. Try gopass init`
3. Then I am trying to init it `gopass init -s test` and get `Error: No useable private keys found`
4. I list my keys with `gpg -K` and then I try to init with `gopass init -s test $myhashkey` and I get "Error: Store is already initialized". So it looks like -s option is being ignored.

I am stuck at this step by now
